udf/histogram_get_sum.sql

```diff
@@ -0,0 +1,39 @@
+/*
+
+Take a list of histograms of type STRUCT<key STRING, value STRING>,
+extract the histogram of the given key, and return the sum value
+
+ */
+CREATE TEMP FUNCTION udf_histogram_get_sum(histogram_list ANY TYPE, target_key STRING) AS (
```

Comment:
I think this should named to indicate that it takes a keyed histogram:
```suggestion
CREATE TEMP FUNCTION udf_keyed_histogram_get_sum(keyed_histogram ANY TYPE, target_key STRING) AS (
```